test/unit/player.test.js

```diff
@@ -2066,6 +2066,23 @@ QUnit.test('player#reset removes remote text tracks', function(assert) {
   log.warn.restore();
 });
 
+QUnit.test('player#reset progress bar', function(assert) {
+
+  let error;
+
+  const player = TestHelpers.makePlayer();
+
+  player.removeChild('controlBar');
```

Comment:
This is a reasonable test, but the `player.controlBar` property appears to still exist even when `removeChild` is called.

```suggestion
  player.removeChild('controlBar');
  player.controlBar = null;
```